CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ Test262 supports the following tags:
  - [**negative**](#negative)
  - [**es5id**](#es5id)
  - [**es6id**](#es6id)
- - [**esid**](#esid)
+ - [**esid**](#esid) (required)
```

Comment:
I'm not sure if we should specify it _here_ saying it's required for all new tests. 

Otherwise it would feel the tests using es5id or es6id only are incorrect. What do you think, @deathbearbrown?